pages/_document.js

```diff
@@ -18,30 +34,30 @@ export default class MyDocument extends Document {
   render() {
     return (
       <html>
-        <Head>
-          <link rel="stylesheet" href="/_next/static/style.css" />
-          {this.props.styleTags}
-          <title>Blockstack</title>
-          {/* Global Site Tag (gtag.js) - Google Analytics */}
-          <script
-            async
-            src={`https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}`}
-          />
-          <script
-            dangerouslySetInnerHTML={{
-              __html: `
+      <Head>
+        <link rel="stylesheet" href="/_next/static/style.css"/>
+        { this.props.styleTags }
+        <title>Blockstack</title>
+        { /* Global Site Tag (gtag.js) - Google Analytics */ }
+        <script
+          async
+          src={ `https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}` }
+        />
+        <script
+          dangerouslySetInnerHTML={ {
+            __html: `
             window.dataLayer = window.dataLayer || [];
             function gtag(){dataLayer.push(arguments);}
             gtag('js', new Date());
             gtag('config', '${GA_TRACKING_ID}');
           `
-            }}
-          />
-        </Head>
-        <body>
-          <Main />
-          <NextScript />
-        </body>
+          } }
+        />
+      </Head>
+      <body>
+      <Main/>
```

Comment:
Minor, but let's keep the indentation here.

Reply:
ran prettier throughout and should be fixed now :)